Kernel/FileSystem/FATFS/Inode.cpp

```diff
@@ -55,7 +57,7 @@ ErrorOr<Vector<BlockBasedFileSystem::BlockIndex>> FATInode::compute_block_list()
     auto fat_sector = TRY(KBuffer::try_create_with_size("FATFS: FAT read buffer"sv, fs().m_device_block_size));
     auto fat_sector_buffer = UserOrKernelBuffer::for_kernel_buffer(fat_sector->data());
 
-    while (cluster < no_more_clusters) {
+    while (cluster < get_end_of_chain_marker()) {
```

Comment:
same comment about get_ here on getters. we don't normally include the implied verb here.

Reply:
Rename squashed into a89da7518bfaa766cdb4fc1b8f9b26d65c7546fa.